### ISSUE TITLE
[Discussion] Starting ship changes and crew members (among others)

### ISSUE THREAD

**Issue by BraedonWooding** [TeamPorcupine/ProjectPorcupine#1751]
- Possibly make it not just single select, so you could have a mining/tourism/trading/raiding base, but your applicant pool size is preset, so choosing multiples dilutes your pool for each
- You pay for the elements of your start so if you take a crappy start on a crappy base with crappy crew, you have tons of money because you didn't spend a thing on any of it
- On a related note, it has also been discussed in the past, to not have all traits automatically known, so you may not know the guy you hired for an engineer is also a pyromaniac, because who puts that on their application... and be able to find these things out in-game (better hire a good psychologist)
- Pool of unrerollable crew, better ones cost more
- Can buy things like uninstalled smelters
- I'm also really fond in general of the idea of having psychology be a legit (and very important) skill  in the game, not only to figure out all of your crew's traits, but instead of the rimworld route of if you don't do a perfect job keeping everyone happy they eventually snap in some random way, you can actually (if you pay attention) send them to the psychologist to get over it.... of course if they're bad they make things worse...  Then there's the risk of your psych having extremely bad traits, but that could be fun, like ending up with a psych who is actually subtly influencing your crew to do all these horrible things
- cthulhu element that causes insanity.  Like the actual lore

**Comment by Dan54** [comment]
I would suggest a moddable directory containing small saves of starting bases. Asteroids would then generate outside this area.

**Comment by BraedonWooding** [comment]
I agree, I also think we have it so that the player gets x amount of money and can build a base before they start the game (and they can choose to spend the money on the base more or people or whatever).  That way maybe two ways to play (well like more than two), a scenario where you play from a set amount of crew and x variables, predefined base, maybe a victory condition and so on... or sandbox where you can build all of these yourself?  What do you think?
